src/test/java/it/pagopa/ecommerce/commons/domain/ConfidentialTest.java

```diff
@@ -100,4 +105,36 @@ void deserializationFailsOnInvalidMetadata() throws InvalidAlgorithmParameterExc
                 () -> objectMapper.readValue(tamperedValue, confidentialEmailTypeRef)
         );
     }
+
+    @Test
+    void roundtripEncryptionDecryptionWithPDVIsSuccessful() throws InvalidAlgorithmParameterException,
+            IllegalBlockSizeException, NoSuchPaddingException, BadPaddingException, NoSuchAlgorithmException,
```

Comment:
Remove the throws declaration here and in other tests since those exceptions are no more thrown by the underling code

Reply:
Fixed in [5072a19](https://github.com/pagopa/pagopa-ecommerce-commons/pull/57/commits/5072a194e3d5caabad8bb88766fb898f9dbf335d)